bucket/flatc.json

```
@@ -1,5 +1,6 @@
 {
     "homepage": "https://google.github.io/flatbuffers/index.html",
+    "description": "An efficient cross-platform serialization library for C++, C#, C, Go, Java, JavaScript, Lobster, Lua, TypeScript, PHP, Python, and Rust.",
```

No need to keep all languages.

```suggestion
    "description": "An efficient cross-platform serialization library for multiple languages.",
```